rpm_spec/subpackages/manageiq-gemset-services

```diff
@@ -0,0 +1,34 @@
+%package gemset-services
+Summary: %{product_summary} Gemset Services
+
+Requires: %{name}-gemset = %{version}-%{release}
+
+%description gemset-services
+%{product_summary} Gemset Services
+
+%post gemset-services
+# Remove any old systemd unit files
```

Comment:
Same comment here as above - whatever you choose there should also go here.

Reply:
:+1: done